Lib/_pyrepl/_module_completer.py

```diff
@@ -42,11 +42,14 @@ def __init__(self, namespace: Mapping[str, Any] | None = None) -> None:
         self._global_cache: list[pkgutil.ModuleInfo] = []
         self._curr_sys_path: list[str] = sys.path[:]
 
-    def get_completions(self, line: str) -> list[str]:
-        """Return the next possible import completions for 'line'."""
+    def get_completions(self, line: str) -> list[str] | None:
+        """Return the next possible import completions for 'line'.
+
+        If 'line' is not an import statement, return None.
```

Comment:
Maybe we could even remove this extra comment? It's pretty clear from the code, but either way is fine :)